src/tests/services/billing-services/load-current-invoice-service.test.js

```diff
@@ -55,6 +55,18 @@ describe('BillingServices', () => {
     expect(result).toEqual(fixtures.formattedResponse)
   })
 
+  it('should return empty object when has no data as result', async () => {
```

Comment:
não adicionar detalhe de implementação ( citação de retornar objeto vazio) no título, adiciona títulos com o comportamento esperado:
should return empty object when has no data as result -> should return correct response for empty data